pulsar-client-cpp/include/pulsar/c/message.h

```diff
@@ -200,6 +200,12 @@ PULSAR_PUBLIC const char *pulsar_message_get_topic_name(pulsar_message_t *messag
 
 PULSAR_PUBLIC int pulsar_message_get_redelivery_count(pulsar_message_t *message);
 
+PULSAR_PUBLIC int pulsar_message_has_schema_version(pulsar_message_t *message);
```

Comment:
Am I correct that this (and other two) functions were never implemented?

Reply:
Yeah, you can open a PR to add the missed implementation.